### ISSUE TITLE
Script just stopped working

### ISSUE THREAD

**Issue by visrosa** [parallelinnovation/Obsidian-Easy-Write-to-Notes-Remotely#1]
I've been using this script extensively (thanks, btw, for the awesome work).

I've done very few modifications to the script, just set my vault and daily note folder and changed instead of returning RoamFormat I return str(date.today()) (all the modifications can be seen in my fork, whatever is there is what I'm actually running).

But since yesterday it stopped working. It can find the daily note alright, but it doesn't seem to recognize the >[[ anywhere. I tried full paths, relative paths, display names and no display names. It just sits there.

Printing indexStart and everythingAfter shows the actual text from the note, so it is reading the right note, it's just not doing anything.

Any clues or tips for debugging?

**Comment by parallelinnovation** [comment]
Hey, I'm glad it's useful! I don't see any changes made to your fork. Where did you change the date format to date.today()? Did you make another function for it? If so, you may have forgotten to replace the CurrentDailyNote() in the RemoveSymbol() function. 

Are there any errors that come up? Also, does it still add the content to the note you sent it to and just keep the symbol there? Thanks

Edit: I updated the code to accommodate your format. Just change the value of DateFormat to 2 if you would like the YYYY-MM-DD format, and let me know how it works.

**Comment by visrosa** [comment]
Wow, thanks a lot for that!

My changes are in the ISO86901-daily-notes branch, should've mentioned that, I'm kind of a git noob, sorry.

Although now I don't need the branch anymore.

I'll do some more testing and post more info shortly.

**Comment by visrosa** [comment]
I was experiencing the bug in my work machine which is where I use your script, so I just tried cloning and running it in my Linux home machine and I'm getting the same.

No errors or weird output. I just get
```
Currently monitoring the daily note: 
/home/void/Documents/lic/2021-11-22/2021-11-22.md
```

It does not add, change or remove anything from any note. It just sits there.

If I `cat` that note name the note shows up, so it is finding the right note. In fact, the last line of the note is `- **19:13**⮞  Teste >[[PE306-2021•PMCDS-RS]]`. 

I'm using the vanilla updated script this time, just added the directories and set date format to 2.

Things I tried: Changing file in visual studio from CRLF to LF, changing it from utf8 to "utf8 with BOM".

**What did work:** Deleting my current daily note, starting from scratch and adding a >[[link]].

When I copy pasted my old note into the new one, the script just stood there again.


I'd be willing to provide the note privately for further investigation.
